e2e/chain/e2e_test.go

```diff
@@ -65,3 +69,23 @@ func TestTransactionStoredOnChain(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, len(txs.Epochs) != 0)
 }
+
+// TestNamespaceSaved ensures that when the stack is running, namespaces can be queried
+// and the address in the namespace can be supplied to a router msg client and send requests.
+func TestNamespaceSaved(t *testing.T) {
+	t.Skip("do not run this test unless the docker containers for base shard, game shard, nakama, and " +
+		"celestia are running. to run the stack, run `make game` and `make rollup`. wait for startup and run this test")
```

Comment:
The dev experience of manually reactivating this test because of an environmental change is not very good. 

Instead this test should be automatic. It should only activate itself if the environment it correct OR the test should be placed at the E2E level where the environment is mocked up. Putting this skip here will pretty much make it unlikely for this thing to ever run EVEN when the set up is correct.

Reply:
yep, i agree. planning to address this here https://linear.app/arguslabs/issue/WORLD-900/make-e2e-tests-with-evm-part-of-the-ci

i would do it now, but launch. can't spend too much time making things pretty unfortunately

Reply:
I already approved. You can ignore all my comments, resolve and have it go up regardless. None of my suggestions should block you at all. 

I suggest putting a todo here as well, in case the ticket gets buried and all context is lost.